server/ai_http.go

```diff
@@ -205,6 +205,12 @@ func handleAIRequest(ctx context.Context, w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	if !orch.CheckModelExists(modelID) {
+		//Orch should log, but don't tell broadcaster the exact error maybe?
```

Comment:
Should we tell the gateway exactly what the error was? (Model folder not found). I think it is enough to tell them the model that failed, but not return the exact error which is mostly useful for the orchestrator